modules/day-time.js

```diff
@@ -0,0 +1,10 @@
+import { DateTime } from './luxon.js';
+
+export default function getTime() {
+  const dt = DateTime.now();
```

Comment:
- [ ] please it would be better if you use understandable names for you variable  instead of `dt` so that other developers can understand you variables what are about